dcrjson/chainsvrcmds.go

```diff
@@ -28,6 +32,16 @@ const (
 	// ANOneTry indicates the specified host should try to connect once,
 	// but it should not be made persistent.
 	ANOneTry AddNodeSubCmd = "onetry"
+
+	// NConnect indicates the specified host that should be connected to.
+	NConnect NodeSubCmd = "connect"
```

Comment:
Please do these in separate blocks next to the type it's defined for so it's clear it is an enum.

```Go
type AddNodeSubCmd string

const(
  ANAdd AddNodeSubCmd = "add"
  ...
)

type NodeSubCmd string

const (
  NConnect NodeSubCmd = "connect"
  ...
)
```

Also, please define it next to the command it applies to.